### DISCUSSION TITLE
Changing desktop resolution on connect with nvidia-settings weird behaviour?

### DISCUSSION THREAD

**Discussion by codevski** [LizardByte/community#157] [Sunshine]
So I'm creating a specific remote desktop shortcut, so it has the correct aspect ratio for my MBP.

Running the command directly (without sunshine `DO`) works perfectly

- Change resolution before connecting with command:
  - `nvidia-settings -a CurrentMetaMode="DPY-3: 2560x1440_144 @2560x1440 +0+0 {ViewPortIn=2560x1600, ViewPortOut=2560x1600+0+0}, DPY-5: 1920x1080_240 @1920x1080 +2560+0 {ViewPortIn=1920x1080, ViewPortOut=1920x1080+0+0}"`
 - connect without any do and undo's, and it works great with the correct aspect ratio
 - Disconnect and run command `nvidia-settings -a CurrentMetaMode="DPY-3: 2560x1440_144 @2560x1440 +0+0 {ViewPortIn=2560x1440, ViewPortOut=2560x1440+0+0}, DPY-5: 1920x1080_240 @1920x1080 +2560+0 {ViewPortIn=1920x1080, ViewPortOut=1920x1080+0+0}"` to set it back

If I put those commands in DO and UNDO, It connects, and my main monitor goes black, and I can see two monitors side by side (I think the main monitor is a snapshot of what it was before it disconnected because I'm not able to move my mouse to it) Disconnecting also doesn't revert, and I need to manually run the command to fix the issue...

Any suggestions on what this could be?

Logs:

```
Feb 01 09:54:56 archlinux sunshine[1348]: [2023:02:01:09:54:56]: Warning: Couldn't get requested display info, defaulting to recording entire virtual desktop
Feb 01 09:54:56 archlinux sunshine[1348]: [2023:02:01:09:54:56]: Info: SDR color coding [Rec. 601]
Feb 01 09:54:56 archlinux sunshine[1348]: [2023:02:01:09:54:56]: Info: Color range: [MPEG]
Feb 01 09:54:57 archlinux sunshine[1348]: [2023:02:01:09:54:57]: Info: Setting default sink to: [sink-sunshine-stereo]
Feb 01 09:54:57 archlinux sunshine[1348]: [2023:02:01:09:54:57]: Info: Found default monitor by name: sink-sunshine-stereo.monitor
```

Also cannot seem to find discord server details... Not on the subreddit or on official docs...

**Comment by ReenigneArcher** [comment]
> Also cannot seem to find discord server details

It's in many places.

- Sunshine web UI (2 places)
- webpage main page -> https://app.lizardbyte.dev (2 places)
- webpage support center -> https://app.lizardbyte.dev/support
- webpage discord page -> https://app.lizardbyte.dev/discord
- docs and project readme point to support center

[a comment by an automated account is not shown]
